frontend/src/features/metadata/api/metadataService.ts

```
@@ -6,18 +6,13 @@ export interface MetadataItem {
 }
 
 export default class MetadataService {
-  getFaculties = async (): Promise<MetadataItem[]> => {
-    const response = await api.get('/metadata/faculty');
-    return response.data;
-  };
-
   getGenders = async (): Promise<String[]> => {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)